requirements.txt

```diff
@@ -12,3 +12,4 @@ PyYAML>=3.11
 requests>=2.10.0
 tinysegmenter==0.3  # TODO(codelucas): Investigate making this >=0.3
 tldextract>=2.0.1
+git+https://github.com/LIAAD/yake.git
```

Comment:
I normally would prefer you make this versioned like the above requirements. but this is OK since it seems the PR owners are all the creators of YAKE.

Is there a reason why this isn't a versioned package from PYPI?